firmware/common2015/utils/logger/logger.hpp

```diff
@@ -61,6 +64,24 @@ extern bool isLogging;
 extern uint8_t rjLogLevel;
 
 /**
+ * [Collects the stream log message into a single string to print]
```

Comment:
Doxygen, the program that converts our code comments into the docs website, doesn't handle square brackets specially, so they'll show up in the text on the site just as they are here.  It would be best to get rid of them.

I noticed that several other comments have these and curly braces in the comments and I think they're an artifact of a comment template snippet that does something like:
~~~{.cpp}
/**
 * @brief [your description here]
 */
~~~

Anyways, I'll make a note to go through and remove them from the other comments throughout our firmware since they're sort of a placeholder.